whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcManagedAttributeDao.java

```diff
@@ -0,0 +1,69 @@
+package net.ripe.db.whois.common.dao.jdbc;
+
+import net.ripe.db.whois.common.aspects.RetryFor;
+import net.ripe.db.whois.common.dao.jdbc.domain.ObjectTypeIds;
+import net.ripe.db.whois.common.domain.CIString;
+import net.ripe.db.whois.common.domain.Maintainers;
+import net.ripe.db.whois.common.rpsl.AttributeType;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.dao.RecoverableDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+
+import static net.ripe.db.whois.common.dao.jdbc.JdbcStreamingHelper.executeStreaming;
+
+@Repository
+@RetryFor(RecoverableDataAccessException.class)
+public class JdbcManagedAttributeDao {
+
+    private final JdbcTemplate jdbcTemplate;
+    private final Maintainers maintainers;
+
+    @Autowired
+    public JdbcManagedAttributeDao(@Qualifier("whoisSlaveDataSource") final DataSource dataSource,
+                                   final Maintainers maintainers) {
+        this.jdbcTemplate = new JdbcTemplate(dataSource);
+        this.maintainers = maintainers;
+    }
+
+    public boolean hasManagedResource(final CIString orgId) {
+        final int orgObjectId = jdbcTemplate.queryForObject(
+            "select object_id from organisation where organisation = ?",
+                new Object[] { orgId },
+                Integer.class
+        );
+
+        return executeStreaming(
+                jdbcTemplate,
+                "select l.object " +
```

Comment:
Uppercase SQL keywords for readibility (and consistency)

Reply:
Ok.